src/features/cookies/helpers/removeGeneratedStorageKey.ts

```diff
@@ -0,0 +1,8 @@
+import { storage } from 'libs/storage'
+
+export const removeGeneratedStorageKey = async (keyToRemove: string): Promise<void> => {
+  const allKeysInStorage = await storage.getAllKeys()
+  if (allKeysInStorage) {
+    allKeysInStorage.forEach((key) => key.startsWith(keyToRemove) && storage.clear(key))
```

Comment:
je trouverais plus facile à lire comme ça


```suggestion
    allKeysInStorage
        .filter((key) => key.startsWith(keyToRemove))
        .forEach((key) => storage.clear(key))
```

Reply:
Carrément !